test/e2e/storage/persistent_volumes-local.go

```diff
@@ -265,6 +265,34 @@ var _ = utils.SIGDescribe("PersistentVolumes-local ", func() {
 				})
 			})
 
+			Context("Set fsGroup for local volume", func() {
+				It("should set fsGroup for local volume", func() {
+					fsGroup := int64(1234)
+					pod, err := createLocalPodWithFsGroup(config, testVol, &fsGroup)
+					framework.ExpectNoError(err)
+					By("Checking fsGroup is correct")
+					_, err = framework.LookForStringInPodExec(config.ns, pod.Name, []string{"ls", "-ld", volumeDir}, strconv.Itoa(int(fsGroup)), time.Second*10)
+					Expect(err).NotTo(HaveOccurred(), "failed: getting the right privileges in the file %v", int(fsGroup))
+				})
+
+				It("should not set fsGroup twice for local volume", func() {
```

Comment:
From the original fsgroup [PR](https://github.com/kubernetes/kubernetes/pull/49988), it was determined that we can't actually enforce that the 2nd pod mount will fail due to different fsgroup.  The best we could do is log a warning.

Reply:
Logs are updated to be more correct. https://github.com/kubernetes/kubernetes/pull/62102/commits/7b1ec6f0eef7a96f97eb3c896f63c3a56bb02d5e

If local volume is already mounted by a pod, [`fsGroup` will be not enforced](https://github.com/kubernetes/kubernetes/pull/49988/files#diff-ec924773446516c2e7bf84c477f5f4ddR288). The second e2e test is used to check this. 

The local plugin [emits a warning event](https://github.com/kubernetes/kubernetes/pull/49988/files#diff-ec924773446516c2e7bf84c477f5f4ddR241), when `fsGroup` cannot be enforced. But event is [only logged by `glog`](https://github.com/kubernetes/kubernetes/pull/49988/files#diff-ec924773446516c2e7bf84c477f5f4ddR60), not reported to apiserver, I cannot verified it in e2e tests. Should we report this event to apiserver, then user can know it by `describe pod`?

Reply:
The intent was for the event to be written to API server.  So that's another bug :(

Reply:
@msau42 OK, I will fix it in another PR.